content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -159,11 +212,11 @@ After fetching the TrueCommand Docker container, enter `docker ps` to see detail
 ![DockerContainerList](/images/TrueCommand/DockerContainerList.png "Finding the TrueCommand Container")
 
 Use the port assigned to the container to access the web interface.
-The list from `docker ps` contains a *PORTS* column.
-Find the port associated with the `ixsystems/truecommand:latest` *IMAGE*.
-The *PORTS* entry is listed as `0.0.0.0:port->80/tcp`, `0.0.0.0:sslport->443/tcp` where `port` and `sslport` are the ports specified earlier.
+The list from `docker ps` contains a **PORTS** column.
+Find the port associated with the `ixsystems/truecommand:latest` **IMAGE**.
+The **PORTS** entry is listed as `0.0.0.0:{port}->80/tcp`, `0.0.0.0:{sslport}->443/tcp` where `{port}` and `{sslport}` are the ports specified earlier.
 
-To access the web interface with no encryption, enter `hostsystemIPaddress:port` in a browser address bar, where `hostsystemIPaddress` is the IP address of the host system that is running the TrueCommand Docker container.
+To access the web interface with no encryption, enter `{hostsystemIPaddress}:port` in a browser address bar, where `{hostsystemIPaddress}` is the IP address of the host system that is running the TrueCommand Docker container.
```

Comment:
only the variable gets italics or the user could think the { } around it are also a variable they can replace with something else. it is proper to say:
where *hostsysteIPaddress* is the IP address of the host system that is running the Truecommand Docker container